symplyphysics/definitions/volumetric_coefficient_of_thermal_expansion.py

```diff
@@ -50,12 +44,10 @@
     volumetric_expansion_coefficient,
     Derivative(volume(temperature), temperature) / volume(temperature),
 )
-r"""
-:code:`alpha_V = 1 / V(T, p) * Derivative(V(T, p), T)`
+"""
+:laws:symbol::
 
-Lambda:
-    .. math::
-        \alpha_V = \frac{1}{V(T, p)} \left( \frac{\partial V}{\partial T} \right)_p
+:laws:latex::
```

Comment:
Same. Also `p` suffix is non-standard. There is no way to autogenerate it like this.